.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=1F36B396-CDA8-4D54-AFB8-A2CB0B84F5EC
```

Comment:
This can't be in the repo

Reply:
The build was failing without it though. Can you make it work without it?

Reply:
Talked with @jvanbaarsen about it. We need the pro license in here, because we're using the design system (which uses the pro license). I'll merge this for now, because our license key was already breached anyway.

Jeroen and I talked about a possible solution for this problem, which we will hopefully implement in the near future.